R/AggregatedDotPlot.R

```diff
@@ -514,6 +542,14 @@ setMethod(".defineDataInterface", "AggregatedDotPlot", function(x, se, select_in
     all_assays <- .getCachedCommonInfo(se, "AggregatedDotPlot")$continuous.assay.names
     all_coldata <- .getCachedCommonInfo(se, "AggregatedDotPlot")$discrete.colData.names
 
+    assay_name <- x[[.ADPAssay]]
+    assay_discrete <- assay_name %in% .getCachedCommonInfo(se, "ComplexHeatmapPlot")$discrete.assay.names
```

Comment:
A discrete assay doesn't even make sense here. How do we compute averages?

Reply:
oops , leftover of copy paste from the heatmap panel
this isn't used anywhere anyway, i'll remove